src/project/internal/notationproject.cpp

```diff
@@ -566,16 +567,17 @@ mu::Ret NotationProject::doSave(const io::path_t& path, engraving::MscIoMode ioM
 
     // Step 1: check writable
     {
-        if (fileSystem()->exists(savePath) && !fileSystem()->isWritable(savePath)) {
+        if ((fileSystem()->exists(savePath) && !fileSystem()->isWritable(savePath))
+            || (fileSystem()->exists(targetContainerPath) && !fileSystem()->isWritable(targetContainerPath))) {
             LOGE() << "failed save, not writable path: " << savePath;
```

Comment:
here better use `targetContainerPath`, the "_saving" is just confusing

Reply:
Also added in new commit.